distribution/ddf-common/src/main/resources/bin/setenv.bat

```diff
@@ -72,13 +77,11 @@ rem As a work around, we will have the ddf_on_error.bat script use the karaf.pid
 rem and it will default to being invoked via the 'script'
 rem The space would be important at the end of the DDF_ON_ERROR value to separate the last argument from
 rem the %p used by the JVM to represent the pid of the JVM
-rem set DDF_ON_ERROR=bin\ddf_on_error.bat script^ 
+rem set DDF_ON_ERROR=bin\ddf_on_error.bat script^
```

Comment:
❗️ the space at the end of that line shouldn't be removed

Reply:
It is a comment.

Reply:
I know it is, but it is part of anther solution to the problem where there is a need for a space at the end of this line as per the comment 2 lines above. 
Either we remove all the comments or we leave them intact

Reply:
I'll add a comment because the IDE wants to remove the space and I don't want to not be able to edit this file in the IDE.